### PR TITLE
Add more common operations to `Quantity`

### PR DESCRIPTION
**Pull request by llucax**
This was extracted from https://github.com/frequenz-floss/frequenz-sdk-python/pull/822/commits/fba4af60ac20b7cd8164b1bede57aa4533a9e234.

### CODE REVIEW

**Comment by llucax** [comment]
Enabled auto-merge.

File: src/frequenz/quantities/_quantity.py [hunk 1]
```diff
@@ -149,6 +149,14 @@ def __round__(self, ndigits: int | None = None) -> Self:
...
         """
         return self._new(round(self._base_value, ndigits))
+
+    def __pos__(self) -> Self:
+        """Return this quantity.
+
+        Returns:
+            This quantity.
+        """
+        return self
```
Comment (Marenz) [inline src/frequenz/quantities/_quantity.py:159]: Blankline after function?

File: src/frequenz/quantities/_quantity.py [hunk 2]
```diff
@@ -139,6 +139,16 @@ def base_value(self) -> float:
...
+    def __round__(self, ndigits: int | None = None) -> Self:
+        """Round this quantity to the given number of digits.
+
+        Args:
+            ndigits: The number of digits to round to.
+
+        Returns:
+            The rounded quantity.
+        """
+        return self._new(round(self._base_value, ndigits))
```
Comment (Marenz) [inline src/frequenz/quantities/_quantity.py:151]: Blankline after function?

File: tests/test_quantities.py [hunk 3]
```diff
@@ -234,6 +234,11 @@ def test_isclose() -> None:
     assert not Fz1(1.024445).isclose(Fz1(1.0))
 
 
+@hypothesis.given(value=st.floats(allow_nan=False, allow_infinity=False))
+@pytest.mark.parametrize("ndigits", [0, 1, 2, 3])
+def test_round(value: float, ndigits: int) -> None:
+    """Test the rounding of the quantities."""
+    assert round(Quantity(value), ndigits) == Quantity(round(value, ndigits))
```
Comment (Marenz) [inline tests/test_quantities.py:241]: Also a blnak line here?

**Review by Marenz — APPROVED** [review]
Ah, it seems the blanklines are correct after all, you just distributed the correct blankline-diffs over different commits

**Comment by llucax** [comment]
Oh, I guess the "stage these lines" from `git gui` lied to me, the diff that showed to me had the correct blank lines. Sorry!

**Pull request merged**